.github/workflows/smoke.yml

```diff
@@ -247,32 +215,51 @@ jobs:
         path: contrib/nydusify/cmd
     - name: Prepare Nydus Environment
       run: |
-        sudo install -D -m 755 contrib/nydusify/cmd/nydusify /usr/local/bin
-        sudo install -D -m 755 target/release/nydusd target/release/nydus-image /usr/local/bin
-        wget https://github.com/containerd/nydus-snapshotter/releases/download/v${{env.SNAPSHOOTER_VERSION}}/nydus-snapshotter-v${{env.SNAPSHOOTER_VERSION}}-x86_64.tgz
-        tar zxvf nydus-snapshotter-v${{env.SNAPSHOOTER_VERSION}}-x86_64.tgz
-        sudo install -D -m 755 nydus-snapshotter/containerd-nydus-grpc /usr/local/bin/
-        sudo wget https://github.com/containerd/nerdctl/releases/download/v${{env.NERDCTL_VERSION}}/nerdctl-${{env.NERDCTL_VERSION}}-linux-amd64.tar.gz
-        sudo tar -xzvf nerdctl-${{env.NERDCTL_VERSION}}-linux-amd64.tar.gz -C /usr/local/bin
-        sudo mkdir -p /opt/cni/bin
-        sudo wget https://github.com/containernetworking/plugins/releases/download/v${{env.CNI_PLUGINS_VERSION}}/cni-plugins-linux-amd64-v${{env.CNI_PLUGINS_VERSION}}.tgz 
-        sudo tar -xzvf cni-plugins-linux-amd64-v${{env.CNI_PLUGINS_VERSION}}.tgz -C /opt/cni/bin
-        sudo install -D misc/benchmark/cni_bridge.conf /etc/cni/net.d/bridge.conf
-        sudo install -D misc/benchmark/nydusd_config.json /etc/nydus/config.json
-        sudo install -D misc/benchmark/containerd_config.toml /etc/containerd/config.toml
-        sudo systemctl restart containerd
-        sudo install -D misc/benchmark/nydus-snapshotter.service /etc/systemd/system/nydus-snapshotter.service
-        sudo systemctl start nydus-snapshotter
-    - name: Prepare Nydus Registry Environment
+        sudo install -D -m 755 misc/benchmark/prepare_env.sh prepare_env.sh
+        sudo ./prepare_env.sh nydus
+        sudo docker pull ${{env.IMAGE}}:${{env.TAG}} && docker tag ${{env.IMAGE}}:${{env.TAG}} localhost:5000/${{env.IMAGE}}:${{env.TAG}}
+        sudo docker push localhost:5000/${{env.IMAGE}}:${{env.TAG}}
+        sudo DOCKER_CONFIG=$HOME/.docker nydusify convert \
+          --source localhost:5000/${{env.IMAGE}}:${{env.TAG}} \
+          --target localhost:5000/${{env.IMAGE}}:${{env.TAG}}_nydus \
+          --fs-version 6 \
+          --oci-ref \
+          --platform linux/amd64,linux/arm64
```

Comment:
It seems we just only run the test on `linux/amd64` platform, `linux/arm64` is unnecessary.

Reply:
> It seems we just only run the test on `linux/amd64` platform, `linux/arm64` is unnecessary.

Actually, we don't need to declaration the platform, so I delete the opt.